modules/demo/client-server/routes/cudf.js

```diff
@@ -0,0 +1,277 @@
+const express     = require('express');
+const {DataFrame} = require('@rapidsai/cudf');
+const path        = require('path');
+const router      = express.Router();
+const fetch       = require('node-fetch');
+const settings    = {
+  method: 'Get'
+};
+
+let gjson         = '';
+let df            = undefined;
+let datasetLoaded = '';
+
+// read Mortgage Dataset from disk using cudf.DataFrame.readCSV, and fetch geoJSON file
+function readMortgageData(callback) {
+  var t0 = new Date().getTime();
+  console.log('readMortgageData');
+  df            = DataFrame.readCSV({
+    header: 0,
+    sourceType: 'files',
+    sources: [path.join(__dirname, '../public/data/m.csv')],
+    dataTypes: {
+      index: 'int16',
+      zip: 'int32',
+      dti: 'float32',
+      current_actual_upb: 'float32',
+      borrower_credit_score: 'int16',
+      load_id: 'int32',
+      delinquency_12_prediction: 'float32',
+      seller_name: 'int16'
+    }
+  });
+  datasetLoaded = 'mortgage';
+  let url =
+    'https://raw.githubusercontent.com/rapidsai/cuxfilter/GTC-2018-mortgage-visualization/javascript/demos/GTC%20demo/src/data/zip3-ms-rhs-lessprops.json';
+
+  fetch(url, settings).then(res => res.json()).then((json) => {
+    gjson = json;
+    console.log('readMortgageData complete');
+    var t1 = new Date().getTime();
+    console.log('Time Taken:', t1 - t0, 'ms');
+    callback(true);
```

Comment:
The node standard is to [pass Errors as the first argument to callbacks](http://fredkschott.com/post/2014/03/understanding-error-first-callbacks-in-node-js/), and if the first argument is `null` or `undefined`, then no error occurred.
```suggestion
    callback(null);
```